composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": "^7.1",
-        "symfony/event-dispatcher": "^3.1 || ^4.0",
+        "symfony/event-dispatcher": "^4.0",
```

Comment:
it should be "symfony/event-dispatcher": "^4.3",

Reply:
sry, I'm getting a little bit careless :P